### DISCUSSION TITLE
Pocketbase to grafana replication

### DISCUSSION THREAD

**Discussion by helander** [maxpert/marmot#107] [Q&A]
I am trying to set up marmot to replicate pocketbase db from one host to another. In the host where pocketbase runs a number of time series are produced and stored in the database. In the other host grafana is running and the intent is for grafana to read data from a replicated version of the pocketbase db. Potentially there may be multiple hosts with grafana. I would like to setup the marmot cluster like this:
- first the pocketbase host marmot instance is started but it has no knowledge about any ”replication hosts” i.e. no information about other host addresses are required at this point. 
- secondly the marmot instance on the grafana host is started, and the address to the marmot instance on the pocketbase host is provided so that the new marmot instance may join the cluster.
- if more grafana hosts are entered they will join the cluster the same way, i.e. by knowing the address of the pocketbase marmot instance

Is it possible to configure the marmot instances to achieve a scenario like this?
If so, how?

**Comment by maxpert** [comment]
Yes it is, in such cases I would recommend running NATS server separately instead of using embedded NATS. That way any marmot instances that come up can point to same NATS server and instances can come and go.
